<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectDate</class>
 <widget class="QDialog" name="selectDate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>97</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="dateBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckAll">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Butfinish">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Go!</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckPlain">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Plain text</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hol text</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckWord">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Word</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CheckChoices">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choices</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
